packages/core/src/components/tooltip/stories/tooltip.args.ts

```diff
@@ -0,0 +1,64 @@
+import { Category } from '@atomium/storybook-utils/enums/table'
+
+export const TooltipStoryArgs = {
+  parameters: {
+    actions: {
+      handles: ['atomClick'],
+    },
+    docs: {
+      description: {
+        component: 'Atom-Tooltip',
```

Comment:
(block) we need to add some description here, you could use another component as example

Reply:
done in [a9fe3bb](https://github.com/juntossomosmais/atomium/pull/367/commits/a9fe3bb09787d994d0fa8dd893ac689c1a1ae128)